modules/telemetry/json_packets.py

```diff
@@ -185,3 +188,167 @@ def parse_mission_file(mission_file: Path) -> MissionEntry:
             length += 1
 
     return MissionEntry(name=mission_file.stem, length=length, filepath=mission_file, version=1)
+
+
+@dataclass
+class TelemetryDataPacketBlock:
+    """A generic block object to store information for telemetry data
+    All stored values must be updated at once!"""
+
+    mission_time: list[int] = field(default_factory=list[int])
+    stored_values: dict[str, list[int]] = field(default_factory=dict)
+
+    def __init__(self, keys: list[str]):
+        """Initializes a TelemetryDataPacketBlock"""
+        self.mission_time = []
+        self.stored_values = dict.fromkeys(keys, [])
+        logger.debug(self)
+
+    def update(self, data: dict[str, int], buffer_size: int) -> None:
+        """Updates the stored values with the given data
```

Comment:
For future docstrings use the Sphinx format; that's what we're standardizing to.